lib/matplotlib/__init__.py

```diff
@@ -1137,7 +1137,14 @@ def interactive(b):
 
 
 def is_interactive():
-    """Return whether to redraw after every plotting command."""
+    """
+    Return whether to redraw after every plotting command.
+
+    .. note::
+
+        This function is only intended for use in backends. End users should
```

Comment:
As far as I understand this, interactive mode does not make sense without `pyplot` (without a figure manager no updates are done anyway).